style.css

```diff
@@ -1,340 +1,926 @@
+    flex-wrap: wrap;
+}
+
+.team__item {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 316px;
+    height: 293px;
+    margin: 0 30px 30px 0;
+    padding: 30px;
+    background-color: #1D1C28;
+}
+
+.team__image {
+    display: block;
+    border-radius: 50%;
+}
+
+.team__name,
+.team__role {
+    text-align: center;
+}
+
+/* Index page About block */
+
+.about__wrapper {
+    max-width: 960px;
+    margin: 0 auto;
+}
+
+.about__title {
+    position: relative;
+    margin-top: 55px;
+    font-family: Bakbak One, Arial, Helvetica, sans-serif;
+    font-size: 24px;
+    line-height: 32px;
+    color: #fff;
+    z-index: 2;
+}
+
+.about__text {
+    font-size: 18px;
+    line-height: 28px;
+    color: #fff;
+}
+
+.about__features {
+    display: flex;
+    flex-wrap: wrap;
+    counter-reset: tidbit-counter;
+    list-style-type: decimal-leading-zero;
+}
+
+.about__features li:before {
+    position: absolute;
+    content: "0" counter(tidbit-counter);
+    counter-increment: tidbit-counter;
+    -webkit-text-stroke: 2px #0265DC;
+    -webkit-text-fill-color: transparent;
+    font-family: Bakbak One, Arial, Helvetica, sans-serif;
+    font-size: 80px;
+    line-height: 96px;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+}
+
+.about__item {
+    position: relative;
+    width: 46%;
+    margin-right: 30px;
+    margin-bottom: 30px;
+    background-color: #1A1925;
+    box-sizing: border-box;
+}
+
+.about__item a {
+    padding: 30px;
+}
+
+.about__item:nth-child(even) {
+    margin-right: 0px;
+}
+
+.main--two-col {
+    max-width: 1280px;
+    margin: 0 auto;
+    display: flex;
+    box-sizing: border-box;
+}
+
+.block__btns {
+    display: flex;
+}
+
+.btn__wrapper {
+    display: flex;
+    text-align: center;
+}
+
+.btn__wrapper--center {
+    justify-content: center;
+}
+
+.logo {
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+    text-decoration-style: none;
+    text-decoration-line: none;
+}
+
+.logo__image {
+    margin-right: 15px;
+}
+
+.logo__text {
+    font-family: Bakbak One, Arial, Helvetica, sans-serif;
+    font-size: 32px;
+    line-height: 42px;
+    text-align: left;
+    color: #ffffff;
+    text-decoration: none;
+    text-decoration-style: none;
+    text-decoration-line: none;
+    text-transform: uppercase;
+}
+
+.social {
+    display: flex;
+}
+
+.social__item {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 40px;
+    height: 40px;
+    padding: 10px;
+}
+
+/* Index page FAQs block */
+
+.faq {
+    width: 960px;
+    margin: 0 auto 40px;
+}
+
+.faq__item {
+    position: relative;
+    margin-bottom: 15px;
+    padding: 24px;
+    background-color: rgba(29, 28, 40, 1);
+}
+
+.faq__question {
+    font-family: Bakbak One, Arial, Helvetica, sans-serif;
+    font-size: 24px;
+    line-height: 32px;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
```

Comment:
For what purpose we usethis styles here?
    text-underline-position: from-font;
    text-decoration-skip-ink: none;

Reply:
i dont know, its from internet`s answer) done